src/main/java/subway/auth/service/AuthService.java

```diff
@@ -0,0 +1,56 @@
+package subway.auth.service;
+
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.stereotype.Service;
+import org.springframework.util.ObjectUtils;
+import subway.auth.dto.TokenRequest;
+import subway.auth.dto.TokenResponse;
+import subway.auth.infrastructure.JwtTokenProvider;
+import subway.common.exception.InvalidLoginException;
+import subway.member.dao.MemberDao;
+import subway.member.domain.Member;
+
+@Service
+public class AuthService {
+    private JwtTokenProvider jwtTokenProvider;
+    private MemberDao memberDao;
+
+    public AuthService(JwtTokenProvider jwtTokenProvider, MemberDao memberDao) {
+        this.jwtTokenProvider = jwtTokenProvider;
+        this.memberDao = memberDao;
+    }
+
+    private boolean checkValidLogin(StringBuilder errorMessageBuilder, String email, String password) {
+        try {
+            if (!memberDao.findByEmail(email)
+                    .getPassword()
+                    .equals(password)) {
+                errorMessageBuilder.append(InvalidLoginException.EMAIL_PASSWORD_MISMATCH);
+                return false;
+            }
+        } catch (EmptyResultDataAccessException erdae) {
+            errorMessageBuilder.append(InvalidLoginException.EMAIL_NOT_EXIST);
+            return false;
+        }
+        return true;
+    }
```

Comment:
여기서 리턴타입은 필요없어 보입니다. StringBuilder 를 넘겨줄 필요도 없고요. 
다양한 exception 이나 에러메시지를 처리하고 싶다면, 상황별로 에러를 throw 해주면 될것 같아요.
```suggestion
    private void checkValidLogin(String email, String password) {
        try {
            if (!memberDao.findByEmail(email)
                    .getPassword()
                    .equals(password)) {
                throw new InvalidLoginException(InvalidLoginException.EMAIL_PASSWORD_MISMATCH);
            }
        } catch (EmptyResultDataAccessException erdae) {
            throw new InvalidLoginException(InvalidLoginException.EMAIL_NOT_EXIST);
        }
    }

Reply:
네 void로 반환형을 변경하고 분기처리에 맞는 예외를 throw하도록 구현하겠습니다.